src/main/scripts/src/app/shared/table/static-table/static-table.component.html

```diff
@@ -0,0 +1,25 @@
+<div>
```

Comment:
Css tables are easier to change based on device, so the table can look slightly different on smaller device screen. Or it can has sticky footer, easier to position additional content (like double border in joined records) and such.

Thank you for pointing out the border-collapse :+1: